src/main/scala/sbtecr/AwsSts.scala

```diff
@@ -7,12 +7,10 @@ import sbt.Logger
 
 private[sbtecr] object AwsSts extends Aws {
 
-  def accountId(region: Region)(implicit logger: Logger): String = {
+  def accountId(region: Region): String = {
     val request = new GetCallerIdentityRequest()
     val response = sts(region).getCallerIdentity(request)
 
-    logger.info(s"AWS account id: ${response.getAccount}")
```

Comment:
Does the behavior change in any way, if we provide an explicit list of `registryIds`? 

The purpose of this function is to return the *active* account id - I'd leave it in for debugging purposes, if possible.

Reply:
I misunderstood that streams can only be used with TaskKey.
But since it was usable without problems, I will return it.